test/unit/specs/store/media-store.spec.js

```diff
@@ -269,22 +270,15 @@ describe('Search Store', () => {
       }
     )
 
-    it('FETCH_SINGLE_MEDIA_TYPE dispatches SEND_SEARCH_QUERY_EVENT', async () => {
+    it('FETCH_SINGLE_MEDIA_TYPE does SEND_SEARCH_QUERY_EVENT', async () => {
       const params = { q: 'foo', shouldPersistMedia: false, mediaType: IMAGE }
       const action = createActions(services)[FETCH_SINGLE_MEDIA_TYPE]
       await action(context, params)
 
-      expect(context.dispatch).toHaveBeenCalledWith(
-        `${USAGE_DATA}/${SEND_SEARCH_QUERY_EVENT}`,
-        {
-          query: params.q,
-          sessionId: context.rootState.user.usageSessionId,
-        },
-        { root: true }
-      )
+      expect(usageData.sendSearchQueryEvent).toHaveBeenCalled()
     })
 
-    it('does not dispatch SEND_SEARCH_QUERY_EVENT if page param is available', async () => {
+    it.skip('does not dispatch SEND_SEARCH_QUERY_EVENT if page param is available', async () => {
```

Comment:
I think it's OK to keep this skipped, the handling of page param should be refactored anyway.

Reply:
The negation for the expected value was not working here but I found the reason, just have to reset the mock after each test, ha!